docs/release-notes/8.3.asciidoc

```diff
@@ -3,8 +3,52 @@
 
 [discrete]
 [[release-notes-8.3.1]]
+
 === 8.3.1
 
+[discrete]
+[[known-issue-8.3.1]]
+==== Known issue
+
+8.3.1 has a bug where detection rules that were created or edited in 8.2 will stop running after you upgrade. Because of this, we advise against upgrading from 8.2 to 8.3.1.
+
+If you already upgraded from 8.2 to 8.3.1, detection rules affected by the bug will have stopped running with an error that is similar to the following example:
+
+[source,text]
+----
+<rule-type>:<UUID>: execution failed - security_exception: [security_exception] Reason: missing authentication credentials for REST request [/_security/user/_has_privileges], caused by: ""
+----
+
+To restore the affected rules and reset their statuses, complete the following.
+
```

Comment:
```suggestion
IMPORTANT: You will need <<enable-detections-ui, manage rules>> privileges.

```
Suggesting we put this here instead of in two places below since it says the same thing in both places.

Reply:
Yeah, I went back and forth on whether it'd be fine to mention once. I assume users will be reading these release notes sequentially, so putting this note before the procedures should be ok.